components/elm/src/biogeophys/CanopyFluxesMod.F90

```diff
@@ -842,18 +858,25 @@ subroutine CanopyFluxes(bounds,  num_nolakeurbanp, filter_nolakeurbanp, &
 
             !! Sakaguchi changes for stability formulation ends here
 
-            rah(p,2) = 1._r8/(csoilcn*uaf(p))
-            raw(p,2) = rah(p,2)
+            rah(p,below_canopy) = 1._r8/(csoilcn*uaf(p))
+            raw(p,below_canopy) = rah(p,below_canopy)
             if (use_lch4) then
-               grnd_ch4_cond(p) = 1._r8/(raw(p,1)+raw(p,2))
+               grnd_ch4_cond(p) = 1._r8/(raw(p,above_canopy)+raw(p,below_canopy))
             end if
 
             ! Stomatal resistances for sunlit and shaded fractions of canopy.
             ! Done each iteration to account for differences in eah, tv.
 
-            svpts(p) = el(p)                         ! pa
-            eah(p) = forc_pbot(t) * qaf(p) / 0.622_r8   ! pa
+            svpts(p) = el(p)                         ! Pa
```

Comment:
@mpaiao , @bishtgautam , others, do you know why we have both svpts and el?  They seem to both be the same thing.

Reply:
By definition one of the variables is declaring that it is a saturated vapor pressure at the leaf surface, and the other is not declared as saturated, but they both act as saturated values.